ha/setup/create_pacemaker_resources.py

```diff
@@ -19,262 +19,289 @@
 from ha.execute import SimpleCommand
 from cortx.utils.log import Log
 
-
-class CreateResourceError(Exception):
-    """Exception to indicate any failure happened during resource creation."""
-
-
-class CreateResourceConfigError(CreateResourceError):
-    """Exception to indicate that given resource configuration is incorrect or incomplete."""
-
+from ha.core.error import CreateResourceError
+from ha.core.error import CreateResourceConfigError
+from ha import const
+
+"""
+# List of resource
+motr-free-space-mon
+s3auth-clone
+    s3auth
+io_group-clone
+    hax-consul
+    hax
+    motr-confd-1
+    motr-ios-1
+    haproxy
+    s3server-1
+    s3server-2
+    s3server-3
+    s3server-4
+    s3backcons
+s3backprod
+sspl-ll-clone
+    sspl-ll
+management
+    kibana
+    csm-agent
+    csm-web
+"""
+
+process = SimpleCommand()
 
 def cib_push(cib_xml):
     """Shortcut to avoid boilerplate pushing CIB file."""
-    cmd_push = f"pcs cluster cib-push {cib_xml} --config"
-    SimpleCommand().run_cmd(cmd_push)
-
+    process.run_cmd(f"pcs cluster verify -V {cib_xml}")
+    process.run_cmd(f"pcs cluster cib-push {cib_xml} --config")
 
 def cib_get(cib_xml):
     """Generate CIB file using pcs."""
-    cmd = f"pcs cluster cib {cib_xml}"
-    SimpleCommand().run_cmd(cmd)
-
+    process.run_cmd(f"pcs cluster cib {cib_xml}")
```

Comment:
Will this overwrite an already existing file?

Reply:
Yes